config/config_test.go

```diff
@@ -141,7 +141,7 @@ func TestFastSyncConfigValidateBasic(t *testing.T) {
 }
 
 func TestConsensusConfig_ValidateBasic(t *testing.T) {
-	// nolint: lll
+	//nolint: lll
```

Comment:
```suggestion
	//nolint: lll
```

I don't _think_ this one is necessary.